packages/gatsby-theme-try-ghost/src/components/common/Layout.js

```diff
@@ -1,11 +1,11 @@
 import React from 'react'
 import PropTypes from 'prop-types'
-import { StaticQuery, graphql, Link } from 'gatsby'
+import { StaticQuery, graphql } from 'gatsby'
 
 import { DocumentHead, StickyNav, SubscribeOverlay, SubscribeSuccess } from '.'
 import { BodyClass } from './helpers'
+import LayoutFooter from './LayoutFooter'
```

Comment:
Please add the new component to index.js, so it can be imported from '.' like "DocumentHead".

Reply:
sure will do